xdsl/dialects/llvm.py

```diff
@@ -783,9 +783,7 @@ class AddressOfOp(IRDLOperation):
     def get(
         global_name: str | StringAttr | SymbolRefAttr, result_type: LLVMPointerType
     ):
-        if isinstance(global_name, str):
-            global_name = StringAttr(global_name)
-        if isinstance(global_name, StringAttr):
+        if isinstance(global_name, (StringAttr, str)):
             global_name = SymbolRefAttr(global_name)
```

Comment:
This is a trivial simplifaction, I hope it's okay to have in this PR?

Reply:
I can not comment, because I don't understand?